WORKSPACE

```diff
@@ -146,9 +147,9 @@ go_register_toolchains()
 
 http_archive(
     name = "build_stack_rules_proto",
-    sha256 = "074dbfbeef97dfb2145cdda0bab6d1d51f459e90e0e835284a59b3880bbebd6e",
-    strip_prefix = "rules_proto-137014a36f389cfcb4987a567b7bd23a7a259cf9",
-    urls = ["https://github.com/stackb/rules_proto/archive/137014a36f389cfcb4987a567b7bd23a7a259cf9.tar.gz"],
+    strip_prefix = "rules_proto-f5d6eea6a4528bef3c1d3a44d486b51a214d61c2",
```

Comment:
buildifier will probably complain that sha needs to be before strip_prefix

Reply:
Ran buildifier which fixed these